attestation-service/docs/claims.md

```diff
@@ -1,14 +1,63 @@
 
-All platforms will by default have two fixed claims:
-- `report_data`: report data when generating the evidence.
-- `init_data`: Hostdata when creating the TEE instance.
+For the general structure of the attestation token, refer to the [EAR specificication](https://datatracker.ietf.org/doc/draft-ietf-rats-ear/).
+
+Generally speaking, the attestation token will include an Appraisal for each device
+that is part of the TCB (including the CPU).
+These appraisals can be accessed through the `submods` field in the EAR Token.
+Each submod is given a generic key constructed from the CPU class and the device count.
+For instance, the CPU submod will be called `cpu0`.
+If a GPU has been attested as part of the guest, there will be a `gpu0` submod.
+If there is more than one GPU, there will be additional `gpuN` submods.
+
+`cpu0` is considered to be the primary attester and has some special
+information associated with it.
+
+# ReportData and InitData
+
+ReportData and InitData are two key Trustee concepts.
+ReportData refers to data provided by a guest at attestation time.
+This is sometimes called user data. The KBS protocol uses this
+field to measure the the nonce and Tee Public Key.
+During attestation Trustee will ensure that the corresponding
+field in the hardware evidence matches the report data values
+that are expected for a given connection.
+
+InitData is a more powerful and subtle concept. See the [InitData Specification](../../kbs/docs/initdata.md)
+for more information.
+The basic idea is that InitData is a generalization over boot-time configuration
+fields such as HostData (on SNP) or MRConfig (on TDX).
+InitData is used to provision dynamic, measured, but not secret 
+configuration data to the guest.
+The InitData plaintext is a TOML or JSON file containing this configuration
+while the InitData hash is the hash of this file which is added to the measuremet.
+A client can optionally provide the InitData plaintext to Trustee.
+If so, Trustee will check the plaintext against the hardware evidence
+and expose the InitData plaintext to the policy engine and as part of the
+attestation token.
+
+Both the InitData and ReportData will usually be included the attestation token.
+These fields will be available under the `AnnotatedEvidence` extension of the
+`cpu0` Appraisal.
+
+If the plaintext InitData is provided, some transformations will be applied
+to the InitData to make it more easy to consume.
+Specifically, if the InitData plaintext is provided as toml,
+some of the data fields in the InitData will be parsed into JSON.
+If the `cdh.toml` or `aa.toml` are present in the InitData,
+they will be parsed as JSON (although the keys will still be called `x.toml`).
+If `policy.rego` is present in the InitData and contains a `policy_data` claim
+at the end of the file, the `policy.rego` field will be removed from the InitData
+and replaced with the `policy_data` parsed as JSON. This will inserted
+with the key `agent_policy_claims`.
```

Comment:
Do you think it make sense to use three items to represent three different special cases (with Markdown hyphens at the beginning of each line) would be clearer and more organized. e.g.

```
Some special keys ...

- `cdh.toml`: if ....
- `aa.toml`: ...
- `agent_policy_claims`: ...
```